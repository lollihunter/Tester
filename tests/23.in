20 18 20
1 1
14 15
12 11
9 8
3 12
3 5
14 13
8 17
8 18
1 8
10 6
6 5
11 14
16 16
16 14
15 7
3 1
2 2
4 9
20 18
