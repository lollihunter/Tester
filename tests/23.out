9
1 18 6 12 4 3 7 15 20 
